UPGRADE.md

```diff
@@ -73,6 +73,142 @@
     +   public function getProvinceCode(): ?string
    ```
 
+1. The constructor of `Sylius\RefundPlugin\Twig\OrderRefundsExtension` has been changed:
+
+    ```diff
+        public function __construct(
+            OrderRefundedTotalProviderInterface $orderRefundedTotalProvider,
+            UnitRefundedTotalProviderInterface $unitRefundedTotalProvider,
+            UnitRefundingAvailabilityCheckerInterface $unitRefundingAvailabilityChecker,
+            OrderRepositoryInterface $orderRepository,
+            RepositoryInterface $refundPaymentRepository
+            RepositoryInterface $refundPaymentRepository,
+    +       RefundTypeFactoryInterface $refundTypeFactory
+        ) {
+            ...
+        }
+    ```
+
+1. The `__invoke` method of `Sylius\RefundPlugin\Checker\UnitRefundingAvailabilityChecker` and `Sylius\RefundPlugin\Checker\UnitRefundingAvailabilityCheckerInterface` has been changed:
+
+    ```diff
```

Comment:
I'm not 100% sure are these diffs useful enough to justify a few additional lines of a file in each point 😄 Maybe simple

```md
`Sylius\RefundPlugin\Checker\UnitRefundingAvailabilityCheckerInterface::__invoke` second parameter has been changed from `RefundType $refundType` to `RefundTypeInterface $refundType`
```

would do the trick? I'm not sure, do we have to mention changes on the class itself, as it's obvious because of interface change :)

Reply:
In these cases, we can just write that we replaced `RefundType` with `RefundTypeInterface` in the given list of classes.